src/main/kotlin/org/jitsi/nlj/stats/EndpointConnectionStats.kt

```diff
@@ -100,13 +100,13 @@ class EndpointConnectionStats(
         if (reportBlock.lastSrTimestamp > 0 && reportBlock.delaySinceLastSr > 0) {
             // We need to know when we sent the last SR
             srSentTimes[SsrcAndTimestamp(reportBlock.ssrc, reportBlock.lastSrTimestamp)]?.let { srSentTime ->
-                // The delaySinceLastSr value is given in 1/65536ths of a second, so divide it by 65.536 to get it
-                // in milliseconds
-                val remoteProcessingDelayMs = reportBlock.delaySinceLastSr / 65.536
-                rtt = receivedTime.toEpochMilli() - srSentTime.toEpochMilli() - remoteProcessingDelayMs
+                // The delaySinceLastSr value is given in 1/65536ths of a second, so divide it by .000065536 to get it
+                // in nanoseconds
+                val remoteProcessingDelay = Duration.ofNanos((reportBlock.delaySinceLastSr / .000065536).toLong())
+                rtt = (Duration.between(srSentTime, receivedTime) - remoteProcessingDelay).toMillis().toDouble()
```

Comment:
Since your RTT is a double here, I'd use something like .toNanos().toDouble() / 1e6.

Maybe we want to add a toDoubleMilli helper function.  I had one on rework-bwe but dropped it as unneeded there:
```
fun Duration.toDoubleMilli(): Double {
    val sec = this.seconds
    val nano = this.nano
    return sec * 1e3 + nano * 1e-6
}
```
in `ClockUtils.kt`

Reply:
ok, pushed a change for this